src/utility/ExportAgent.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+const rimraf = require('rimraf');
+const Dialogflow = require('dialogflow');
+const AdmZip = require('adm-zip');
+
+const projectId = process.env.DIALOGFLOW_PROJECT_ID;
+const file = process.argv[2] || './Agent';
+
+async function exportAgent() {
+  rimraf.sync(file);
+  const agentClient = new Dialogflow.v2.AgentsClient({
+    keyFilename: process.env.GOOGLE_APPLICATION_CREDENTIALS,
+  });
+  const [res] = await agentClient.exportAgent({ parent: projectId });
+  const buffer = res.result.agentContent;
+  const zip = new AdmZip(buffer);
+  zip.extractAllTo(file);
+}
+
+exportAgent();
```

Comment:
Nice nice. Not really necessary but I think I would be cool if you could pass parameters to the script like 
`--project-id` and `--google-credentials-path`.

Reply:
I initially had that but removed it in favor of using the credentials and ID from dotenv.
Would you say I should use the value from dotenv as the default and overwrite it if arguments are given?